koku/masu/processor/expired_data_remover.py

```diff
@@ -130,18 +130,13 @@ def remove(self, simulate=False, provider_uuid=None, line_items_only=False):
             ([{}]) List of dictionaries containing 'account_payer_id' and 'billing_period_start'
 
         """
-        removed_data = None
+        removed_data = []
         disable_purge_line_item = (Provider.PROVIDER_AZURE, Provider.PROVIDER_AZURE_LOCAL, Provider.PROVIDER_OCP)
         no_data_msg = "%s has no line item data to be be removed."
+        line_items_only = False
```

Comment:
We can remove the `line_items_only` argument from `ExpiredDataRemover.remove` and the other places throughout the code.

This if block can also be removed since we no longer have to remove line item data as part of the summary processing:
https://github.com/project-koku/koku/blob/2087dcce7761656bd04f02c191f0f3ea474aa671/koku/masu/processor/tasks.py#L429

When trying your branch locally it seems to be removing all summary data because of this.

Reply:
Doing that test I could definitely see data being removed from the partition tables but I still see the tables with psql (`reporting_awscostentrylineitem_daily_summary_2021_06`). Should these be removed after the table drop?

Reply:
The table drop should eliminate the table partition. I will double-check the code here.